guides/03_API_Overview.md

```diff
@@ -38,6 +38,10 @@ Using the above objects, create a [DefaultMeetingSession](https://aws.github.io/
 const meetingSession = new DefaultMeetingSession(configuration, logger, deviceController);
 ```
 
+### 1e. Create a meeting from your server application
```

Comment:
Sorry, I just realized we can keep this before 1c because, we use these responses in 1c. Should have commented in the earlier review, sorry about that.

Reply:
No problem. that make sense. Will change.